trees/avltree/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2017 Benjamin Scher Purcell <benjapurcell@gmail.com>
```

Comment:
@spewspews is it OK with you if I append this license file into the [root licence](https://github.com/emirpasic/gods/blob/master/LICENSE)?

```
Copyright (c) 2015, Emir Pasic
All rights reserved.

Redistribution and use in s...

-------------------------------------------------------------------------------

AVL Tree

Copyright (c) 2017 Benjamin Scher Purcell <benjapurcell@gmail.com>

Permission to use, copy, modify, and distribute this software...
```